services/vision/mlvision/detector.go

```diff
@@ -0,0 +1,170 @@
+package mlvision
+
+import (
+	"bufio"
+	"context"
+	"image"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/nfnt/resize"
+	"github.com/pkg/errors"
+
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/utils"
+	"go.viam.com/rdk/vision/objectdetection"
+)
+
+const (
+	// UInt8 is one of the possible input/output types for tensors.
+	UInt8 = "uint8"
+	// Float32 is one of the possible input/output types for tensors.
+	Float32 = "float32"
+)
```

Comment:
Move this into ml_model.go, rather than in detector

Reply:
Done